crates/sui-adapter/src/adapter.rs

```diff
@@ -497,37 +495,33 @@ pub fn generate_package_id(
     modules: &mut [CompiledModule],
```

Comment:
Pre-existing on this PR, but I think we need to fix the code surrounding module re-serialization. Probably have an old github issue for it somewhere...

Reply:
I think it's Issue #69, referenced in a TODO comment above.